### PR TITLE
Added like and dislike feature in Issue

### PR DESCRIPTION
**Pull request by Krishna1922**

### CODE REVIEW

File: home/views.py [hunk 1]
```diff
@@ -1,14 +1,15 @@
 from django.shortcuts import render, HttpResponseRedirect, reverse, HttpResponse, redirect
-from django.core.mail import EmailMessage, send_mail, BadHeaderError
+from django.core.mail import send_mail, BadHeaderError
 
 from home.helpers import EmailThread
 from django.core import mail
-from project.models import Project, Issue, IssueAssignmentRequest, ActiveIssue, PullRequest, Domain, SubDomain
+from project.models import Project, Issue, IssueAssignmentRequest, Like
```
Comment (hackko-20) [inline home/views.py:6]: These can be imported together why divide it?

**Comment by hackko-20** [comment]
You also need to add 'thumbs up' 'thumbs down' icon with every issue card, clicking on which will like/dislike for that user + issue accordingly

File: home/views.py (hunk 1 above)
Comment (Krishna1922) [reply home/views.py:6]: actually previously it was together but when i running it was giving flake8 error so that why i have put in second line

**Comment by Krishna1922** [comment]
@hackko-20 Sorry for that i was forget to update index.html file

**Comment by Krishna1922** [comment]
image

File: home/views.py (hunk 1 above)
Comment (Krishna1922) [reply home/views.py:6]: previously line 6 and 8 was together but giving flake8 error so i divide in some parts

Comment (hackko-20) [reply home/views.py:6]: Try to divide it using newline, not separating imports into two parts, it is generally not a good practice.

File: .github/workflows/django.yml [hunk 2]
```diff
@@ -2,9 +2,9 @@ name: Django-Automated-Testing
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "my-patch" ]
```
Comment (hackko-20) [inline .github/workflows/django.yml:5]: I don't think this file should be in the PR, it should go into .gitignore. Please check once.

**Comment by hackko-20** [comment]
Resolve the comments, they are small only. Rest looks good to me.

**Comment by Krishna1922** [comment]
@hackko-20 Done mam

**Comment by hackko-20** [comment]
LGTM.

**Pull request merged**